src/resolvers/invitation.resolvers.ts

```diff
@@ -140,7 +155,23 @@ const invitationResolvers: IResolvers = {
             })),
             orgToken
           });
-          const result = await sendInvitationEmail(email, {role, org, userId, orgToken});
+
+          // Create a token for the invitation
+          const token = jwt.sign(
+            {
+              name:org.name,
+              email,
+              role:role,
+            },
+            SECRET,
+            { expiresIn: '2d' }
+          );
+        
+          const newToken = token.replace(/\./g, '*');
+          newInvitation.invitationToken = newToken
+          const link = `${process.env.REGISTER_FRONTEND_URL}/${newToken}`;
+        
+          const result= await sendInvitationEmail(email, org.name,link)
```

Comment:
and this why the comes in twice ??

Reply:
I’ve resolved the issue by creating a reusable function for token generation, ensuring the process is only executed once.